source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,94 @@
+---
+title: "Core Concepts"
+description: "How Home Assistant works."
+---
+
+Home Assistant, like all home automation software, allows you to control smart devices in your home. This page will explain the parts of Home Assistant and how they relate.
+
+## Devices
+
+In Home Assistant, a **device** is a physical object in your connected home. An example would be a lightbulb, a Chromecast, or a smart hub.
+
+Home Assistant has a **discovery** feature, which scans for new devices as they’re added and adds them to your Home Assistant automatically.
+
+**Device trackers** are devices like cell phones that can move around.
+
+## Entities
+
+Because many devices do more than one thing, most things in Home Assistant use  **entities**. An entity is a single function or property of a device, like a single sensor, light, or switch.
+
+A device can provide multiple entities. For example, a device called “Sam’s Phone” might have entities like location, battery level, and step count. Some devices like lightbulbs only have a single entity, and devices like hubs may have no entities at all.
+
+Some entities are useful for controlling things in your home:
+
+- **Switches** let you check if something is switched on or off?
+- **Sensors** can be a wide range of things that watch for changes in the environment, like motion sensors and temperature sensors
+- **Binary sensors** detect something that is either on or off
+- **[Scripts](https://www.home-assistant.io/docs/scripts/)** are special entities that run custom code when they’re triggered
+
+Other entities are there to respond to Home Assistant:
+
+- **Lights** can turn on and off, and sometimes dim and change color
+- **Covers** are things that can move up and down, like garage doors and smart blinds
+
+Some entities are part of a **domain**, or a built-in category of entities that are managed in a similar way. For example, there is a “lights” domain in Home Assistant.
```

Comment:
All entities have a domain don't they?

`entity_id: domain.object_id`

Reply:
Yep, entities are always in a domain.

Reply:
also `are managed` is a passive voice, try making it active.

Reply:
fixed passive voice!

Good to know that all entities have a domain. What's an object id? I'll take a stab at clarifying but it will probably be wrong.

Reply:
In the example entity id `switch.electric_boogaloo`  the domain is `switch` and the object id is `electric_boogaloo`

Reply:
fixed!